Bio/Blast/NCBIWWW.py

```diff
@@ -19,9 +19,9 @@
 
 from io import StringIO
 from Bio._py3k import _as_string, _as_bytes
-from Bio._py3k import urlopen as _urlopen
-from Bio._py3k import urlencode as _urlencode
-from Bio._py3k import Request as _Request
+from urllib.request import urlopen as _urlopen
```

Comment:
Why the underscores?

Reply:
i thought of removing them but i wanted to consult here before doing that.

Reply:
Let's discuss this on #2483